deploy/olm-catalog/service-telemetry-operator/manifests/infra.watch_servicetelemetrys_crd.yaml

```diff
@@ -149,6 +149,72 @@ spec:
                       type: object
                   type: object
               type: object
+            logs:
+              description: Logs related backend configuration.
+              properties:
+                loki:
+                  description: Logs storage backend Loki
+                  properties:
+                    enabled:
+                      description: Enable Loki as a storage backend for logs
```

Comment:
Can we just change this description to say `---TESTING ONLY--- Enable Loki as a storage backend for logs` ?

Reply:
Alternatively we could downstream patch this out during build, but that feels heavy?